packages/sui-js/src/pipe/index.js

```diff
@@ -1,3 +1,5 @@
 const pipe = (...funcs) => arg => funcs.reduce((acc, value) => value(acc), arg)
-
 export default pipe
+
+export const asyncPipe = (...funcs) => arg =>
```

Comment:
I don't really understand the point of using this "helpers", don't you know your functions?

I mean, in case of `asyncPipe[bla, bla2, bla3]` and `bla().then(bla2).then(bla3)` everybody will have to check what asyncPipe is the first time but everybody knows what the second example does. Also, in case something changes and you need to add a second value to any of these functions you'r screwed and have to use thens anyways, don't you? Maybe I'm missing something

Reply:
The result of `asyncPipe[bla, bla2, bla3]` and `bla().then(bla2).then(bla3)` is the same, and you are right that people may check what asyncPipe do. Nevertheless, In motor we need to chain some fn (async mappers) and it improves the code readability. @carlosvillu suggest us to make this async pipe.

Reply:
I would argue that pipe is a very standard function. Everyone knows what a pipe is and how it works. AsyncPipe is just a detail in how it runs a normal pipe. I don't think it would be really hard to grasp.